build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -165,6 +167,8 @@ CONFIG_AMEBASMART_WIFI=y
 # Realtek RTL8730E BLE Support
 #
 CONFIG_AMEBASMART_BLE=y
+# CONFIG_FTL_ENABLED is not set
+CONFIG_BLE_INDICATION=y
```

Comment:
I just search `CONFIG_BLE_INDICATION` and find it's amebad's config, not amebasmart.
I think this adding is wrong.

Reply:
Understood, removed